client/core/core.go

```diff
@@ -6401,19 +6401,22 @@ func (c *Core) cacheRedemptionFeeSuggestion(t *trackedTrade) {
 		}
 		c.log.Debugf("unable to retrieve fee rate from FeeRater. falling back to other methods: %v", err)
 	}
-	// Try to find any book that might have the fee.
+	// Check any book that might have the fee recorded from an epoch_report note
+	// (requires a book subscription).
 	redeemAsset := t.wallets.toAsset.ID
 	feeSuggestion := t.dc.bestBookFeeSuggestion(redeemAsset)
 	if feeSuggestion > 0 {
 		atomic.StoreUint64(&t.redeemFeeSuggestion, feeSuggestion)
 		return
 	}
 	// Don't request it if we already have one.
+	// TODO: declare the rate stale at some point and fetch a new one.
 	if atomic.LoadUint64(&t.redeemFeeSuggestion) != 0 {
 		return
```

Comment:
After considering this line for far too long, I'm pretty sure this can become an issue with a non-`FeeRater` wallet in a headless client (that probably does not have an `orderbook` subscription to receive `epoch_report` notes that power `bestBookFeeSuggestion`).  It would only do the `fee_rate` request once and never get another one, forever.

Reply:
It does look like it can become stale. Need to store a time stamp too?

Reply:
Created issue https://github.com/decred/dcrdex/issues/1515